tools/distrib/python/grpcio_tools/grpc_tools/protoc.py

```diff
@@ -58,6 +58,8 @@ def _maybe_install_proto_finders():
                     ProtoFinder(_SERVICE_MODULE_SUFFIX,
                                 _protoc_compiler.get_services)
                 ])
+                sys.path.append(
+                    pkg_resources.resource_filename('grpc_tools', '_proto'))
```

Comment:
Directly appending the `sys.path` might have unwanted side-effects. Do we have other options?

Reply:
This is how the import mechanism for runtime protos works in general. There really isn't any other option here. I think the risk is low because:

 - We also have complete control over what's included in this directory -- only proto files.
 - This addition only happens in the case that runtime protos are enabled. Note that we're also updating the meta_path in this branch.

Reply:
Since runtime protos are turned on by default now, it means all users (>=3.5) importing gRPC will incur this `sys.path` change. I guess preload well-known protos might be too costly, so there isn't a win-win solution. As you mentioned, we can point users to the environment variable if they have trouble about this change. LGTM.